.classpath

```diff
@@ -13,7 +13,7 @@
 			<attribute name="test" value="true"/>
 		</attributes>
 	</classpathentry>
-	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.7">
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.6">
 		<attributes>
```

Comment:
co cie sklonilo do wyboru eclipse zamiast intellij?

Reply:
ofc nie mowie ze to zle, zastanawiam sie tylko gdzie sie uzywa eclipse, w C++? niestety w javie industry standard to bedzie intellij